src/components/shared/epic.tsx

```diff
@@ -14,7 +14,8 @@ import { body, headline } from '@guardian/src-foundations/typography';
 import { SvgArrowRightStraight } from '@guardian/src-icons';
 import { ThemeProvider } from 'emotion-theming';
 import { acquisitionsClient } from 'native/nativeApi';
-import React, { useEffect, useRef, useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
+import * as React from 'react';
```

Comment:
As above, are you sure this is needed?

Reply:
@JamieB-gu I'd be interested to know why you weren't getting problems with the `Fragment`s, but they were showing up on @dskamiotis's machine. Commenting here so that I can keep up to date on any discussion around that.

Reply:
What kind of errors were you seeing? Lint, compile or runtime?

Reply:
I believe runtime (during npm run test)

Reply:
I believe it was `ReferenceError: React is not defined`. I think that there's potentially some sort of problem with how Webpack and Babel are transpiling the `Fragment`s, though I'm not sure how or why.

Reply:
Oh it's a _test_ failure? Right that makes sense, the new config needs to be applied to Jest as well. I've pushed a commit to this PR that should fix things.